src/KineticsStripOutputNode.cpp

```diff
@@ -0,0 +1,102 @@
+#include "KineticsStripOutputNode.h"
+#include <iostream>
+#include <algorithm>
+
+// Reusable state for sending data packets
+static int out_fd = -1;
+// static struct sockaddr_in out_addr;
+
+// This is dynamic because we need to allocate memory based on
+// the number of pixels per strip.
+unsigned char out_packet[512] = {};
```

Comment:
Can you move these into the class?

You will have to make a `KineticsStripOutputNodePrivate` class and use it to for data storage:

```
class KineticsStripOutputNodePrivate : public SelfTimedReadBackOutputNodePrivate {
public:
    KineticsStripOutputNodePrivate(Context *context); // add additional constructor arguments if you wish
```

Once you do that, you'll also need to create a few boilerplate methods:
```
public:
    KineticsStripOutputNode(const KineticsStripOutputNode &other);
    KineticsStripOutputNode *clone() const override;
private:
    QSharedPointer<KineticsStripOutputNodePrivate> d() const;
```

Take a look at another node type (such as ImageNode) to see what should go into these methods. They are all pretty straightforward.

Sorry that there's so much strange boilerplate, I'm vaguely working on improving it.